tests/test_request.py

```diff
@@ -2,7 +2,7 @@
 from .testing import mockserver
 import pydu.request
 from pydu.network import get_free_port
-from pydu.request import FileName, check_connect, update_query_params
+from pydu.request import FileName, check_connect, update_query_params,cookies_string_to_dict
```

Comment:
Same. You was missing blank between `update_query_params,` and `cookies_string_to_dict `

Reply:
Update below:
* Use `TypeError` , `ValueError` instead of `RequestException `. 
* Use `for-loop` instead of `list comprehensions`.
*  Style be more `Pythonic`.

Thank you for your advice ,  and it has been optimized .